conda_build/build.py

```diff
@@ -150,6 +150,22 @@ def create_info_files(m, files, include_recipe=True):
             else:
                 shutil.copy(src_path, dst_path)
 
+    license_file = m.get_value('about/license_file')
+    if license_file:
+        filenames = 'LICENSE', 'LICENSE.txt', 'license', 'license.txt'
+        if license_file is True:
+            for fn in filenames:
+                src = join(source.get_dir(), fn)
+                if isfile(src):
```

Comment:
Can we print a comment to stdout here about which file we are going to use for the license file?